Server/resources/accounts/duelyst.account.json

```diff
@@ -289,6 +289,51 @@
                 "hasCombo": false,
                 "remainingNumber": 20
             },
+            {
+                "name": "Azurite Lion",
```

Comment:
The PR title implies that you're only adding one card, but youre adding 3 (azurite lion, emerald rejuv, and sun wisp

Reply:
This is a change to the duelyst account.  And it's a consequence of my script adding cards alphabetically.

So basically, in this PR this POSITION of Azurite has changed, but the card is actually already in master. (same with sun wisp)